transformer-runner/src/runner.ts

```diff
@@ -40,7 +42,7 @@ const runtime = new Runtime(
 );
 
 export async function initializeRunner() {
-	console.log(`[WORKER] starting...`);
+	logger.info(`starting...`);
```

Comment:
Minor: `starting` -> `Starting`. I am not sure there is a convention though, noticed this elsewhere.